### PR TITLE
Fixed Some Maria/InnoDB Issues and Enhanced dbench

### PR DESCRIPTION
**Pull request by chabad360**

### CODE REVIEW

**Comment by chabad360** [comment]
the next 50% of commits are mostly updates to dbench

**Comment by chabad360** [comment]
I've been working on it so there will be many commits added over time

**Comment by Thunderbottom** [comment]
can you just post the dbench and readme changes in one PR?

File: test.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def print_out(val,delay):
 		elif val == 2:
 			try:
 				global r
-				r = requests.get("http://site1.local:8000")
+				r = requests.get("http://localhost:8000")
```
[the remark this answers is not in the document]
Comment (chabad360) [reply test.py]: If I remember correctly it was after I had migrated the compose file to V3, for some reason it wasn't connecting to the site, so I changed it and it was resolving (but that might not be why, again, I don't remember why I did some things the way I did)

**Pull request closed without merging**